.github/workflows/scheduled-tests.yml

```diff
@@ -48,3 +52,10 @@ jobs:
       run: |
         source .venv/bin/activate
         pytest --expensive-tests
+
+    - uses: JasonEtco/create-an-issue@v2
```

Comment:
Is there any way for us to pass in more information, like what commit the test was running at, or any error messages?

Reply:
I do not believe the scheduled workflow as a concept of what commit it is currently at, but since we have the git repo, we can extract the commit at HEAD from that. I'll look into passing in some additional info and circle back. Thanks.

Reply:
This has been update to this: https://github.com/mlcommons/newhelm/issues/242
Thanks for the suggestion. More useful this way